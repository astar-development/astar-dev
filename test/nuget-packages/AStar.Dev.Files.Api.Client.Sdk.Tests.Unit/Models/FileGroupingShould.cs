using AStar.Dev.Utilities;

namespace AStar.Dev.Files.Api.Client.Sdk.Models;

[TestSubject(typeof(FileGrouping))]
public class FileGroupingShould
{
    [Fact]
    public void ContainTheExpectedProperties()
        => new FileGrouping { FileSize = 1234, Height = 123, Width = 456 }.ToJson().ShouldMatchApproved();
}
