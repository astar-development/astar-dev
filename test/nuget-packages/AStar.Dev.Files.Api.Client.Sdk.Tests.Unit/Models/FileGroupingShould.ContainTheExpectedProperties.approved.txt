{
  "fileSize": 1234,
  "height": 123,
  "width": 456
}